buildSrc/src/main/java/azure/AzureBlobStorage.kt

```diff
@@ -103,9 +103,9 @@ class AzureBlobStorage(private val containerName: String) {
     val blob = blobContainer.getBlockBlobReference(blobName)
 
     println("Upload $path to $blobName")
-    Files.newInputStream(path).use {
-      blob.upload(it, Files.size(path))
-    }
+//    Files.newInputStream(path).use {
```

Comment:
I don't get this, could you please elaborate?

Reply:
One thing that we should keep in mind with PR: commit messages are not really visible :)
In this case, I temporary deactivate the publication on Azure, so we can test the new release process and check if there is no issue.
I added a comment in the code to explain this. Of course, I will reactivate this after the test.